eng/emitter-package.json

```diff
@@ -2,7 +2,7 @@
   "name": "typescript-emitter-package",
   "main": "dist/src/index.js",
   "dependencies": {
-    "@azure-tools/typespec-ts": "0.18.0",
+    "@azure-tools/typespec-ts": "0.19.0",
```

Comment:
@minhanh-phan could you help update the relevant lock file also? Otherwise we would fail to generate considering the mismatch between them.

https://github.com/Azure/azure-sdk-for-js/blob/main/eng/emitter-package-lock.json

Reply:
Thank you for the reminder. [The PR](https://github.com/Azure/azure-sdk-for-js/pull/27870#pullrequestreview-1744980376) got merged!